src/libcollections/slice.rs

```diff
@@ -1410,6 +1410,11 @@ impl<T> [T] {
 
     /// Converts `self` into a vector without clones or allocation.
     ///
+    /// The resulting vector can be converted back into a box via
+    /// the [`into_boxed_slice`] method.
+    ///
+    /// [`into_boxed_slice`]: vec/struct.Vec.html#method.into_boxed_slice
```

Comment:
unfortunately, I have a feeling this link won't work because this method appears on both the `slice` and `Vec` doc pages. as is, the link works on the `slice` page, but on the [vec page](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.into_vec), it would end up linking to `std/vec/vec/struct.Vec.html#method.into_vec`. we'll find out soon whether this passes the linkchecker on Travis CI, but i'm guessing this link will probably need to be removed